code/game/machinery/doors/airlock_types.dm

```diff
@@ -557,8 +501,8 @@
 /obj/machinery/door/airlock/hatch/cockpit
 	icon = 'icons/obj/doors/almayer/dropship1_pilot.dmi'
 	name = "\improper Cockpit"
-	req_access_txt = "22"
-	req_one_access_txt = "0"
+	req_access = list(ACCESS_MARINE_DROPSHIP)
+	req_one_access = list()
```

Comment:
```suggestion
	req_one_access = null
```

Reply:
I changed it for now. If you can agree on how you want it finally, let me know.